testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ContactListAdapter.kt

```diff
@@ -24,17 +25,23 @@ class ContactListAdapter(private val contactListener: ContactListener) : Recycle
         bindNonVisibleView("Id: ", contact.id, holder.contactId)
         bindNonVisibleView("Name: ", contact.name, holder.contactName)
         bindNonVisibleView("Email: ", contact.email, holder.contactEmail)
+        bindNonVisibleView(
+            "Optional Email(s): ",
+            contact.allEmailList?.joinToString(),
```

Comment:
Probably, we can also check if `contact.allEmailList` is empty to hide `holder.contactOptionalEmail` view to avoid the following UI:

![Screen Shot 2022-12-28 at 13 02 15](https://user-images.githubusercontent.com/66930373/209755449-9fa4d7d1-b4d6-480f-bdda-e9fffd5cb2f7.png)

Reply:
I just thought about it but that would be inconsistent on the part of the field. I'm also ok with this.

Reply:
Yeah sure, what you feel okay, no problem 👍